app/routes/cms/index.tsx

```diff
@@ -27,8 +27,8 @@ export default function Index(): JSX.Element {
 	const { firstname, lastname } = useLoaderData<LoaderData>();
 
 	return (
-		<Center minW="100vw" minH="100vh">
-			<Heading>
+		<Center h="100%" w="100%">
```

Comment:
Looks like you accidentally replaced the `Center` props with the same bad values again (see #164):

> On a small viewport, this will cause content clipping, instead of making it horizontally scrollable

Reply:
Oh yeah sorry, i meant to write `minH=100%`and `minW=100%`